### DISCUSSION TITLE
New Payment Methods proposed for Bisq (Please comment on the ones you would like added)

### DISCUSSION THREAD

**Discussion by pazza83** [bisq-network/bisq#5287] [Q&A]
## New Payment Methods proposed for Bisq

Hi following discussion with Bisq users the following payment methods are being considered for inclusion in Bisq.

Not all payment methods will be added as we will be looking to add the ones that will be most widely used. Therefore if you see a payment method you would like to use please comment on this thread saying what would would like to see, or even better comment on the GitHub issue.

The more feedback received the better we can develop Bisq to be useful to people like you,

### Multi-Currency Payment Methods

- [Paxum](https://github.com/bisq-network/growth/issues/235)
- [Jeton](https://github.com/bisq-network/growth/issues/234)
- [Paysera](https://github.com/bisq-network/growth/issues/233)
- [Skrill](https://github.com/bisq-network/growth/issues/232)
- [CelPay](https://github.com/bisq-network/growth/issues/231)
- [WorldRemit](https://github.com/bisq-network/growth/issues/230)

### Indian Payment Methods

- [NEFT](https://github.com/bisq-network/growth/issues/236)
- [RTGS](https://github.com/bisq-network/growth/issues/237)
- [UPI](https://github.com/bisq-network/growth/issues/225)
- [PayTM](https://github.com/bisq-network/growth/issues/226)
- [IMPS](https://github.com/bisq-network/growth/issues/224)

### African Payment Methods

- [M-Pesa](https://github.com/bisq-network/bisq/issues/5022)

### US Payment Methods

- [ATH Movil](https://github.com/bisq-network/growth/issues/238)
- [TransferWise (USA)](https://github.com/bisq-network/growth/issues/217)
- [Strike](https://github.com/bisq-network/growth/issues/220)
- [Domestic Wire Transfer](https://github.com/bisq-network/growth/issues/218)
- [ACH](https://github.com/bisq-network/growth/issues/219)

### European Payment Methods

- [Bizum](https://github.com/bisq-network/growth/issues/222)
- [Monese](https://github.com/bisq-network/growth/issues/227)
- [Satispay](https://github.com/bisq-network/growth/issues/213)
- [Verse](https://github.com/bisq-network/growth/issues/223)
- [iDeal](https://github.com/bisq-network/proposals/issues/244)

### Colombian Payment Methods

- [Efectly](https://github.com/bisq-network/growth/issues/192)
- [Daviplata](https://github.com/bisq-network/growth/issues/194)
- [Nequi](https://github.com/bisq-network/growth/issues/193)

### Brazilian Payment Methods

- [CapCode](https://github.com/bisq-network/growth/issues/215)
- [PIX](https://github.com/bisq-network/growth/issues/208)

### Gift Voucher Payment Methods

- [BitRefill](https://github.com/bisq-network/growth/issues/239)

**Comment by pazza83** [comment]
Also, SWIFT has already been confirmed to be added as an international / multi-currency payment method and is with the devs now.

**Reply by Conza88** [reply]
I was about to rage 😆 .

**Comment by flix1** [comment]
MTN Mobile Money (https://www.worldremit.com/en/mtn-mobile-money) is used very widely in a dozen african countries. It has almost 100M users. It is the most used payment method on Localbitcoins for these countries. See for example https://localbitcoins.com/country/GH.

**Reply by RififiCastorjunior** [reply]
Pazza have already those payments in the radar array if i'm not mistaken.

**Comment by m52go** [comment]
I think UPI should be a top priority as it could enable the Indian market.

**Reply by pazza83** [reply]
Thanks, I agree it seems like it could be popular and much needed.

**Comment by Conza88** [comment]
### **Multi-Currency Payment Methods**
**Paxum** - By P2P transfer to other Paxum Users | $0.25 cost. Recieving money: By Local Bank Transfer from 3rd party | 0.5%, min. $5.00.
**Jeton** - oof, 2% fee's. 
**Paysera** - Payment between users is free. Withdrawals to banks are free or low cost. $15 transaction fee. 
**Skrill** - 40M users ;o. Free for international transfers (using independent exchange rate provided by Reuters). 2% fee for sending money domestically. As a take only, given lots of fee's for other stuff (skrill to skrill etc. avoid that). But how you want it setup, this is the way!
**CelPay** - meh. 
**WorldRemit** - Have tested. Sneak in fees via not a great exchange rate. You still see what is being sent prior to, but its not obvious that they take their cut there. Everything else looks low. Anything for big amounts draws q's. I got funds through with my 'reason'. 

TL;DR my ordering:

1. Skrill seems very promising, also most users by far?
2. Paxum / World Remit
3. Paysera
4. Jeton
5. Celpay

**Comment by jimmyneutront** [comment]
I will be very interested in offering liquidity in the BTC/USD market if ACH is supported.

**Comment by m52go** [comment]
@pazza83 I suggest reducing the feedback window for adding payment methods. Is there anything in particular you're looking for / waiting for?

If a payment method is technically a good fit for Bisq (e.g., hard to charge back, works well with dispute resolution, etc) and makes sense to add (in major market, or a market with potential) then I think it should just be added. Adding new payment methods isn't technically intensive so the bar for adding doesn't need to be so high.

**Reply by pazza83** [reply]
Hi @m52go

Thanks for the feedback. I would like to make the process as quick as possible.

I proposed the [Proposed payment method addition process and estimated timelines #206](https://github.com/bisq-network/growth/issues/206) here and am keen to meet the proposed dates for all payment methods.

The idea of the 60 day window is as follows:

- Give users opportunity to feedback about a particular payment method (The reality is though that this might be too long. Most activity occurs immediately after posting)
- Giving myself enough time to review the payment methods and recommend them to be added / not added to Bisq. I would like to review all payment methods this cycle to make sure I do not recommend anything that might not be a good fit for Bisq. There are ~28 to review so this takes a bit of time.

The 60 day is also followed by another cycle of 30 days to then give the arbitrators / mediators the time to respond to any proposed additions. I think this is an important step as there was some different opinions in the past about the feasibility of mediation for certain payment methods.

Once the above is complete I would also like to batch the payments for addition according to region / country. I think this will make quicker in terms of communication with devs and also easier to market the new payment methods to a particular user group (fiat currency).

I think the process I have chosen to follow means that things will progress more slowly initially but the end result will be more relevant payment methods are added quicker and hopefully uptake will be improved.

For example I know you are keen to add UPI, as am I, but I would like to batch together multiple payment methods. For example see recent discussion here: https://www.reddit.com/r/BitcoinIndia/comments/mogpj0/any_bisq_users_from_india_here_follow_up/

**Reply by m52go** [reply]
Personally I think the feedback timeline is too long, but I wasn't very effective at this role so I won't advise one way or another. 

I'm not sure batching is necessary...in liquid markets with lots of payment methods, like USD and EUR, just 1-2 tend to account for 80-90% of trades anyway. In cases like UPI (clear user indication) and KWD (willing and enthusiastic promoter on-the-ground) I'm not sure waiting is ideal. Enthusiasm can waver and promoter might get discouraged. 

For new markets with multiple good payment methods, adding in waves could be beneficial, as there can be new posts and new headlines as new ones are added.

**Comment by pazza83** [comment]
Hi @m52go thanks for the feedback.

I will try and speed up the timeline by requesting input from mediators / arbitrators earlier on in the cycle. I think most of the times this will just be a formality. Hopefully there will be much less discussion needed for the current payment methods being considered than CBM / SWIFT etc.

I think batching is a good way of utilizing the limited resources Bisq has available, but agree that it would be good if the process were speeded up to reduce the impact of any dampers to enthusiasm.

For payment methods in India I think adding multiple payment methods will be better than adding one.  UPI for example, while likely to be popular would be limited to buying/selling 0.025 btc.  Other payment methods would be needed to allow for larger trade sizes. Therefore volume traded may skew away from UPI to another payment method with higher limits.
